### DISCUSSION TITLE
Xmrig keeps bringing my windows systems down

### DISCUSSION THREAD

**Discussion by Yamanipanuchi** [RainbowMiner/RainbowMiner#1683] [Q&A] [answered]
I've had this happen on several Windows 10 PC's now. System becomes almost unusable and when I look to see what's being mined, Its and algo using Xmrig.

Dont know if its one of my setting, rbm's setting or Xmrig itself.

Some of these machines i've seen this on are freshly installed.

Attached is the logs from one of the machines.

**Comment by Yamanipanuchi** [comment]
[debug_2021-04-29.zip](https://github.com/RainbowMiner/RainbowMiner/files/6401796/debug_2021-04-29.zip)

**Comment by RainbowMiner** [comment, marked as answer]
It looks like you have CPU mining enabled, with 
```
  "CPUMiningThreads": 4,
  "CPUMiningAffinity": "0xf",
```
in config.txt. This uses all CPU threads for mining, so no more air to breathe for anything else.

I recommend leaving at least one thread unused. In fact, there is an automatic fix build-in. Just set:
```
  "EnableAutoAdjustAffinity": "1",
```

RainbowMiner will then do the rest.

**Comment by Yamanipanuchi** [comment]
Yup, I see that. Had it enable at some point and clearly ended up disabled again... Thanks!
